Sources/SwiftGtk/enums/RecentSortType.swift

```diff
@@ -0,0 +1,48 @@
+import CGtk
+
+/// Used to specify the sorting method to be applyed to the recently used resource list.
+/// 
+/// https://docs.gtk.org/gtk3/enum.RecentSortType.html
+public enum RecentSortType {
+    /// Do not sort the returned list of recently used resources.
+    case none
+
+    /// Sort the returned list with the most recently used items first.
+    case mru
```

Comment:
It would be good for readability to rename this to `mostRecentlyUsed` same for `lru`